weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/callsTableQuery.ts

```diff
@@ -123,6 +90,72 @@ export const useCallsForQuery = (
   }, [callResults, calls.loading, total]);
 };
 
+export const useDownloadFilterSortby = (
```

Comment:
that makes this `download` specific? Seems like `useCallsForQuery` could use this?